weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx

```diff
@@ -108,11 +108,9 @@ export const PlaygroundPageInner = (props: PlaygroundPageProps) => {
         setPlaygroundStateFromTraceCall(callWithCosts.result.traceCall);
       }
     }
-  }, [
-    callWithCosts.loading,
-    setPlaygroundStateFromTraceCall,
-    callWithCosts.result,
-  ]);
+    // Only set the call the first time the page loads, and we get the call
```

Comment:
this was rerunnning every render because of callwithCosts.result

Reply:
hmmm, but isn't `callwithCosts.result` expensive? shouldn't that not be changing?